### DISCUSSION TITLE
AWS CLI is overriding my theme

### DISCUSSION THREAD

**Discussion by 0xgokuz** [ohmyzsh/ohmyzsh#12372] [Help] [answered]
![Screenshot 2024-04-21 at 14 29 26](https://github.com/ohmyzsh/ohmyzsh/assets/155065525/ba23fea4-4c82-4873-9bf5-8f58224299a1)

How can I activate my theme? I want it to look like this:

![bullet-train](https://camo.githubusercontent.com/9720e5a0044a812a107eec10eb59fd0d3e2a8df1b44668d0af42bd5464bb65f1/687474703a2f2f7261772e6769746875622e636f6d2f6361696f676f6e64696d2f62756c6c65742d747261696e2d6f682d6d792d7a73682d7468656d652f6d61737465722f696d672f707265766965772e676966)

**Comment by mcornella** [comment, marked as answer]
Looks like the theme in the screenshot is bullet-train. Search and follow the steps for enabling it. https://github.com/caiogondim/bullet-train.zsh
